docs/docs/guides/metadata.md

```diff
@@ -0,0 +1,106 @@
+#  Add metadata to your application
+
+This guide will walk you through the usage of Context Metadata in Keptn.
+
+After reading this guide you will be able to: 
+- add metadata to applications, workloads
+and tasks in two ways.
+- add those same information also in traces
+
+## Introduction
+
+`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
+and links to traces for a specific application.
+This enables you to enrich your Keptn resources and your traces with additional
+information, making it easier to understand and analyze
+the performance of your applications.
```

Comment:
```suggestion
Content metadata is additional information 
that can be added to your `KeptnApp` resource and links

This enables you to enrich your Keptn resources and your traces with additional
information, making it easier to understand and analyze
the performance of your applications.
Metadata is added with the
`KeptnAppContext` custom resource.
```
Move this to the very beginning (minus the "Introduction" header, then follow it with your intro to this guide.

Reply:
I think we need to rework this again, the message is not 100% correct: metadata can be added as annotation to a deployment/any workload or using a Context CRD I should highlight that in the intro to make sense of the two following sections